source/component/PasDoc_Parser.pas

```diff
@@ -2180,6 +2181,12 @@               { include "identifier = class of something;" as standard type }
               Exit;
             end;
           end;
+        KEY_TYPE: begin
```

Comment:
This test seems deceptively simple :) You detect type helpers simply when seeing `type`  keyword, you do not check for `helper` keyword as far as I see.

Did you test whether it doesn't cause problems for non-type helpers that start with `type`? Like this:

```
type
  TMyInteger = type Integer;
```

This is valid Pascal, and it is not a type helper.

Possibly you need to do more checks, i.e. you should check that after `type` you really have `helper` keyword, and only then go with CIO_TYPEHELPER parsing.